src/test/java/core/basesyntax/service/FileWriterImplTest.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.FileReader;
+import core.basesyntax.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FileWriterImplTest {
+    private static final String CSV_OUTPUT
+            = "src/test/java/core/basesyntax/resources/finalReportTest.csv";
+    private static final String CSV_OUTPUT_EXPECTED
+            = "src/main/java/core/basesyntax/resources/finalReport.csv";
+    private static FileWriter fileWriter;

```

Comment:
you should use separate test files. Don't use files from the base package for testing